### ISSUE TITLE
Can I stop bubbling?

### ISSUE THREAD

**Issue by gongshun** [mauricius/vue-draggable-resizable#136]
I used a nested tag，When I dragged the child element, the parent element was also dragged.

**Comment by gongshun** [comment]
As shown[
![1](https://user-images.githubusercontent.com/29347231/50319876-8b132e00-04c1-11e9-8f34-1aee24d52a83.gif)
](url)

**Comment by TitanFighter** [comment]
There is such prop as [active](https://github.com/mauricius/vue-draggable-resizable#active) (and [draggable](https://github.com/mauricius/vue-draggable-resizable#draggable)) and there is such event as [activated](https://github.com/mauricius/vue-draggable-resizable#activated). Try to `active="false"` the parent element when the nested element is activated. Something like:

```
<parentElement :active="isParentActive"></parent>
<nestedElement `@activated="isParentActive = false"</nestedElement>

data () {
  return {
    isParentActive: false
  }
}
```

**Comment by gongshun** [comment]
> 有[活动](https://github.com/mauricius/vue-draggable-resizable#active)（和[可拖动](https://github.com/mauricius/vue-draggable-resizable#draggable)）这样的道具，并且有[激活](https://github.com/mauricius/vue-draggable-resizable#activated)这样的事件。`active="false"`激活嵌套元素时尝试使用父元素。就像是：
> 
> ```
> <parentElement :active="isParentActive"></parent>
> <nestedElement `@activated="isParentActive = false"</nestedElement>
> 
> data () {
>   return {
>     isParentActive: false
>   }
> }
> ```
Thank you for your answer, I've tried, but Sometimes it's not work
```
<vue-draggable-resizable 
    :x="contentsStyle.x" 
    :y="contentsStyle.y" 
    :w="contentsStyle.width" 
    :h="contentsStyle.height"
    :active="parentDrag" 
    :draggable="parentDrag" 
    :resizable="parentDrag" 
    @resizing="parentDrag=true" 
    @dragging="parentDrag=true" 
    :parent="true"  
    @activated="parentDrag=true" 
    @deactivated="parentDrag=false">
    <vue-draggable-resizable 
        v-for="(item,index) in contentChildren" 
        :x="item.x" 
        :y="item.y" 
        :w="item.width" 
        :h="item.height" 
        :key="index" 
        @resizing="parentDrag=false"
        @dragging="parentDrag=false" 
        @activated="parentDrag=false" 
        @deactivated="parentDrag=true" 
        :parent="true">
        <p class="no-select">{{item.name}}</p>
    </vue-draggable-resizable>
</vue-draggable-resizable>
```
![1](https://user-images.githubusercontent.com/29347231/50340660-cfc7b500-0513-11e9-9031-15413883fb3e.gif)

**Comment by TitanFighter** [comment]
In few hours I will fix your code. Right now very busy.

**Comment by TitanFighter** [comment]
I tried my own code and also Mauricius's code from #81, but it does not work. In #81 I asked Mauricius for a help.

**Comment by gongshun** [comment]
> 我尝试了自己的代码以及来自[＃81的](https://github.com/mauricius/vue-draggable-resizable/issues/81) Mauricius的代码，但它不起作用。在[＃81](https://github.com/mauricius/vue-draggable-resizable/issues/81)我问毛里求斯寻求帮助。

thank you , I modified the source code to solve this problem and succeeded.
![image](https://user-images.githubusercontent.com/29347231/50389036-328b9d00-071c-11e9-93ea-7d943b8fa1ca.png)
vue-draggable-resizable.vue

**Comment by mauricius** [comment]
Yes, stopping the propagation of events works in your case, but unfortunately many other cases require events to fully propagate. I already made the mistake of adding and removing the `stop` event modifier before, and each time there were issues with some particular cases. For example see https://github.com/mauricius/vue-draggable-resizable/issues/114 and https://github.com/mauricius/vue-draggable-resizable/issues/127, but I'm sure that there are others.

Stopping event propagation is dangerous, (see [this wonderful arictle on CSS-Tricks](https://css-tricks.com/dangers-stopping-event-propagation/) for a better explanation), therefore I decided to remove all the event modifiers from the code.

I'm planning a feature for version 2 that will eventually solve that problem too, if you are still interested.

**Comment by a14d** [comment]
@mauricius does it work now on version 2 ?
